pkg/utils/utils.go

```diff
@@ -13,6 +20,25 @@ func UniqueStrings(ss []string) []string {
 	return results
 }
 
+func UniqueLibraries(libs []types.Library) []types.Library {
+	var uniqLibs []types.Library
+	unique := map[string]struct{}{}
+	for _, lib := range libs {
+		h := hash(lib)
```

Comment:
`fmt.Sprintf("%s@%s", lib.Name, lib.Version)` seems enough. `lib.ID` is better, but all structs doesn't have it.

Reply:
Agree 👍